google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +184,18 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def get_schema_update_actions():
+        """Maintain all the schema changes for this table.
+
+        Returns:
+            dict: A mapping of Action: Column.
+        """
+        column_mapping = {'CREATE': [Column('resource_name',
```

Comment:
Instead of `column_mapping`, it would document better if the naming is `schema_update_mapping` or `schema_action_mapping`

Reply:
Updated